### ISSUE TITLE
'\uXXXXXXXX' needs capital U in Characters-and-strings.md

### ISSUE THREAD

**Issue by gpoore** [JuliaDocs/Julia-Cheat-Sheet#47]
This is a helpful Julia overview! I noticed in `Characters-and-strings.md`, under "Any UTF character", that 8-digit HEX `'\uXXXXXXXX'` should actually be `'\UXXXXXXXX'` (capital U).

**Comment by brakmic** [comment]
Thanks for pointing this out.
Best, create a Pull request so we can merge and (and you will get a proper contribution entry in our logs).

Cheers,

**Comment by abelsiqueira** [comment]
I think is best to leave unresolved issues open and only close when the corresponding PR is merged.

**Comment by brakmic** [comment]
No problem. I this case I expected the original poster to open a Pull Request and continue the discussion there. But let's leave this one open. 😄 

Regards,

**Comment by gpoore** [comment]
For something this insignificant, I don't need a contribution entry. Unfortunately, I don't have time for a pull request.

**Comment by brakmic** [comment]
ok

**Comment by waldyrious** [comment]
@brakmic, @abelsiqueira's point is still valid even if @gpoore wasn't available to submit the change themselves. I've now submitted a PR at #50 which, once merged, would indeed fix this issue.

Please bear in mind that having issues closed in this manner might actually discourage the community from contributing, rather than having the opposite effect (as I assume to be your intention).

If you _do_ want to force people to only submit code contributions (which I [wouldn't recommend](https://github.com/kentcdodds/all-contributors)), a better way to do that, IMHO, would be to disable issues in this repo, rather than closing issues after they've been created.

**Comment by brakmic** [comment]
@waldyrious 
No worries, I have no such intentions. It's just open source, so everyone can contribute.

I am actually not that much involved in this little project. Long time ago I published it under my GitHub profile where it remained "invisible" for quite some time. Then it got discovered by one of the Julia community members who asked me if I could move it to JuliaDocs. Now we're here. 

And because there are many people who contribute lots of new things I don't think I should do much except click on "merge" from time to time. 😅 

So, in short: whatever you guys & girls find "OK" is "OK" for me as well 😄 

I'll now reopen this thread and wait for the community to decide when we should close it.

Best regards and thanks for all your work!
